test/karma/eventDrops.js

```diff
@@ -80,4 +80,18 @@ describe('d3.chart.eventDrops', () => {
         test(false, false);
         test(true, true);
     });
+
+    it('should be possible to supply objects as data', () => {
+        const div = document.createElement('div');
+        const data = [
+            { name: 'foo', dates: [{date: new Date('2010-01-01'), foo: 'bar1'}]},
```

Comment:
Perhaps should we rename the `dates` to `data`, as it is not an array of dates anymore.